src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  background-image: url(../images/header-M.jpg);
```

Comment:
Consider adding a fallback background color for browsers that do not support background images